src/pages/are-you-sure.tsx

```diff
@@ -47,23 +47,6 @@ export const AreYouSure: FunctionComponent<AreYouSureProps> = ({
   );
 };
 
-export const buildAreYouSureQuery = (
-  action: string,
-  yes: string,
-  no: string,
-  consequences: string
-): string => {
-  return (
-    "?" +
-    new URLSearchParams({
-      action,
-      yes,
-      no,
-      consequences,
-    }).toString()
-  );
-};
-
```

Comment:
As @matthew-a-carr pointed out, this was unnecessary.  Replace with simple re-use of `formatUrlQueryParams`